test/test-sauce-labs.js

```diff
@@ -0,0 +1,222 @@
+/* jshint esversion: 6 */
```

Comment:
Yup, it's time I think https://blog.codinghorror.com/rule-of-three/ want me to tackle that as part of this PR?